facebookincubator/facebook-for-woocommerce
# Revamp notices

# Summary

Removes FBE 1.5 interim notices and adds our new notice for when synced products are disabled.

### Story: [CH 55150](https://app.clubhouse.io/skyverge/story/55150)
### Release: #1345 

## Details

When products are set to "Do not sync" after previously having sync enabled, display a message to the user that they should consider deleting the product(s) manually and a link for how to do so.

The notice only shows on one page load, and is permanently dismissible.

## UI Changes

- New notice: https://cloud.skyver.ge/JrubJX29

## QA

- Access token & catalog ID stored (can be any)

1. Set a product's "Facebook Sync" setting to "Sync and..." and save
    - [x] No notice is displayed
1. Set the setting to "Do not sync" and save
    - [x] The new notice is displayed
1. Dismiss the notice
1. Set the setting to "Sync and..." and save
1. Set the setting back to "Do not sync" and save
    - [x] No notice is displayed
1. Delete the `_wc_plugin_framework_facebook_for_woocommerce_dismissed_messages` option from the database
1. From the products list table screen, select a few products and use the bulk action to "Include" in facebook sync
    - [x] No notice is displayed
1. Use the bulk action to exclude a few products
    - [x] The notice is displayed
1. Dismiss the notice
1. Use the bulk action to exclude a few products
    - [x] No notice is displayed

- [x] Acceptance tests pass

## Before merge
